### PR TITLE
Updating Google readme files to include mandatory disclosure.

### PR DESCRIPTION
**Pull request by malexanderlim**
### <samp>🤖 Generated by Copilot at 4296909</samp>

This pull request adds a disclaimer to the README files of all components that use Google APIs, stating that Pipedream complies with the Google API Services User Data Policy. This is a requirement from Google to access and transfer data from their APIs. The affected components are `google_ads`, `google_appsheet`, `google_calendar`, `google_classroom`, `google_cloud_translate`, `google_cloud`, `google_dialogflow`, `google_directory`, `google_docs`, `google_drive`, `google_photos`, `google_recaptcha`, `google_search_console`, `google_sheets`, `google_slides`, `google_tasks`, and `google_workspace`.

### <samp>🤖 Generated by Copilot at 4296909</samp>

> _`README` files change_
> _Google API disclaimer_
> _A winter mandate_

## WHY

In order to go through the verification process for Google apps, they are requesting that we add a disclosure to the app's page.

### <samp>🤖 Generated by Copilot at 4296909</samp>

*  Add a disclaimer to the README files of all Google-related components to comply with the Google API Services User Data Policy ([link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-b5a988bd24150363c8486d1094429b86709ffd23d7fa9154dcb60e08e63b66d5L21-R21), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-b9b0732a0b2277fcc8b2fbe6ac131535e38253fe983b3d336fda7a9d5bf89ee3L9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-80228f2269b474599737868e97424f527491b678f69e962c54c2669e333c3d2bL21-R21), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-a4eb0b25201de4501a0c43f20efac21df44200e838bc9d63a27383f4cc439affL18-R18), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-0cfe1db25ccd6d1501e811f9d2c4cbc2b068658a02bfef58f986240e160ca565L8-R8), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-e1c66df5b30389cb4bb211817a1a816c3a470a4ef5de058322725585c863c6eeL11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-25862e89f184c3fb15b02065f088f4ee7dc1cecfa5363d3a96f36b2498f182abL15-R15), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-70272a397022f10926214fbb294723d5bc3a1d328429a4a537e29f8b13024e20L10-R10), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-beb1f302a5705c1c460039ed2797ebd86c081bdf108a3c10e3da4f8b107635e0L15-R15), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-abae7fb42a427ce275d39a1c8e70810cb25f5d633e0e0e4e02c68b6ab1088e8dL11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-9d33755cdd4503d7d0e6f9a892877351ac1c7dd526ba5ebab55fce0c1d693f01L16-R16), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-7044e23bd8ec4368ea637e5e6e5f4c66d5e17dadb2a10573a6633b6cf79a7501L16-R16), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-768f1c446c9c427a0d8e77f23ee1f5c545d26c09abf0287b5baaeb922e416379L11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-dbe2807899f74a10022890b1eb9cc91c15e6f2dfd4fdf109fb48aed38bc0a2aaL10-R10), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-2025f7841132c53798d03898355a0ddef4cb3a6183487fc49927a7e9eee07af5L14-R14), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-7ed1e240799e9aa709515c8e9d106acce89d0d8f17d4ddafbde2ca100e78d7f2L8-R8), [link](https://github.com/PipedreamHQ/pipedream/pull/6963/files?diff=unified&w=0#diff-5221e89bfaccce95710586df55fd614e425be901b059bf2557156aafd7336d8fL12-R12))

### CODE REVIEW

File: components/google_ads/README.md [hunk 1]
```diff
@@ -18,3 +18,5 @@ and campaigns programmatically. With the API, you can automate common tasks,
 such as creating and managing campaigns, adding and removing keywords, and
 adjusting bids. You can also use the API to get information about your
 campaigns, such as campaign stats, keyword stats, and ad performance.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_ads/README.md:22]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_ads/README.md:22]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_appsheet/README.md [hunk 2]
```diff
@@ -6,3 +6,5 @@ computers. With Appsheet, you can easily create apps that allow you to track
 and manage your data, create reports and visualizations, and even share your
 data with other users. Appsheet is also easy to use, with a simple interface
 that makes it easy to get started.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_appsheet/README.md:10]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_appsheet/README.md:10]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_calendar/README.md [hunk 3]
```diff
@@ -18,3 +18,5 @@ include:
   to see their tasks and events in one place.
 - A reminder application that uses Google Calendar data to remind users of
   upcoming events.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_calendar/README.md:22]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_calendar/README.md:22]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_classroom/README.md [hunk 4]
```diff
@@ -15,3 +15,5 @@ API:
   assignments for their classes
 - A student submission system that allows teachers to view and grade student
   submissions
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_classroom/README.md:19]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_classroom/README.md:19]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_cloud/README.md [hunk 5]
```diff
@@ -5,3 +5,5 @@ services from their own applications. Services that can be accessed include
 Google Cloud Storage, Google Cloud Datastore, Google Cloud Functions, and
 Google Cloud Pub/Sub. With the Google Cloud API, developers can build a variety
 of applications that take advantage of Google Cloud services.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_cloud/README.md:9]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_cloud/README.md:9]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_cloud_translate/README.md [hunk 6]
```diff
@@ -8,3 +8,5 @@ Cloud Translate API include:
 - A translation app that can translate documents or webpages from one language
   to another.
 - A chat app that can translate messages between users in different languages.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_cloud_translate/README.md:12]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_cloud_translate/README.md:12]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_dialogflow/README.md [hunk 7]
```diff
@@ -12,3 +12,5 @@ Dialogflow can be used to build a variety of applications, including:
 - Voice assistants
 - Customer service applications
 - Robotic process automation (RPA) applications
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_dialogflow/README.md:16]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_dialogflow/README.md:16]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_directory/README.md [hunk 8]
```diff
@@ -7,3 +7,5 @@ With the Google Directory API, you can:
 - Update a user's name, email address, and other profile information
 - Change a user's password
 - Suspend or delete a user
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_directory/README.md:11]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_directory/README.md:11]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_docs/README.md [hunk 9]
```diff
@@ -12,3 +12,5 @@ including:
 - Web scraping tools
 - Text analysis tools
 - And much more!
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_docs/README.md:16]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_docs/README.md:16]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_drive/README.md [hunk 10]
```diff
@@ -8,3 +8,5 @@ Using the Google Drive API, you can build applications that:
 - Search for files
 - Track changes to files
 - And much more!
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_drive/README.md:12]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_drive/README.md:12]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_photos/README.md [hunk 11]
```diff
@@ -13,3 +13,5 @@ download photos and videos; and search for photos and videos.
 Albums, photos, and videos can be shared with other users, and users can add
 comments to photos and videos. The API also provides a set of tools for
 developers to build photo-sharing applications.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_photos/README.md:17]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_photos/README.md:17]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_recaptcha/README.md [hunk 12]
```diff
@@ -13,3 +13,5 @@ Here are some examples of what you can build using the Google reCAPTCHA API:
 - A simple CAPTCHA form that can be used to protect your website from spam
 - An advanced CAPTCHA form that can be used to protect your website from more
   sophisticated attacks
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_recaptcha/README.md:17]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_recaptcha/README.md:17]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_search_console/README.md [hunk 13]
```diff
@@ -8,3 +8,5 @@ applications. Here are a few examples:
 - A dashboard that visualizes data fetched from the Google Search Console API.
 - A tool that allows users to input a query and receive results from the Google
   Search Console API in real-time.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_search_console/README.md:12]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_search_console/README.md:12]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_sheets/README.md [hunk 14]
```diff
@@ -7,3 +7,5 @@ Some examples of things you can build using the Google Sheets API include:
   source
 - A tool that generates graphs and charts from data in a Google Sheet
 - A service that sends data from a Google Sheet to another API or application
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_sheets/README.md:11]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_sheets/README.md:11]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_slides/README.md [hunk 15]
```diff
@@ -11,3 +11,5 @@ Using the Google Slides API, you can build applications that do the following:
 - Crop images
 - Add or remove comments
 - And more!
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_slides/README.md:15]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_slides/README.md:15]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_tasks/README.md [hunk 16]
```diff
@@ -5,3 +5,5 @@ With the Google Tasks API, you can build applications that
 - Create new tasks
 - Read, update, and delete existing tasks
 - View task lists
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_tasks/README.md:9]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_tasks/README.md:9]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

File: components/google_workspace/README.md [hunk 17]
```diff
@@ -9,3 +9,5 @@ Google Workspace account. For example, you can:
 - Suspend or delete user accounts
 - View audit logs
 - Generate reports
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```
Comment (dylburger) [inline components/google_workspace/README.md:13]:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment (dylburger) [reply components/google_workspace/README.md:13]: Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).

**Review by dannyroosevelt — APPROVED** [review]

**Pull request merged**